README.Mozilla.md

```diff
@@ -0,0 +1,85 @@
+Quick startup:
+==============
+
+All work will be done in ~/DeepSpeech/. Please update accordingly.
+This has been tested successfully on:
+* Ubuntu 14.04 with TITAN X GPU
+* Ubuntu 16.04 with K1000M GPU 
+* Debian Sid (as of september/october 2016) with GTX 970 GPU
+
+If you use different versions of CUDA/CUDNN please update accordingly below,
+CUDA version 8.0 and CUDNN version 5.1 are assumed.
+
+Compute capabilities should cover most usecase. If not, please refer to NVIDIA
+website: https://developer.nvidia.com/cuda-gpus and update the list with your
+needs.
+
+CUDA/cudnn path should be normalized (i.e., no "../" component) otherwise build
+system will not be happy.
+
+You might need an account on NVIDIA's developper area to download cudnn
+
+# System setup
+
+* install bazel in ~/DeepSpeech/bin/: just use the shell script installer at
+  https://github.com/bazelbuild/bazel/releases
+* install CUDA in ~/DeepSpeech/CUDA/ from https://developer.nvidia.com/cuda-toolkit-archive
+ * ``PERL5LIB=. sh cuda_VERSION_linux.run --silent --verbose --override --toolkit --toolkitpath=$HOME/DeepSpeech/CUDA/``
+* install cudnn in ~/DeepSpeech/CUDA/ from https://developer.nvidia.com/rdp/cudnn-download
+ * ``tar xvf cudnn-CUDA_VERSION-linux-x64-CUDNN_VERSION.tgz --strip-components=1 -C $HOME/DeepSpeech/CUDA/``
+* ``cd DeepSpeech && virtualenv tf-venv && tf-venv/bin/pip install numpy scipy python_speech_features && cd ..``
+* ``git clone https://github.com/mozilla/tensorflow --branch warpctc && cd tensorflow``
+* For ld, either:
+ * ``export LD_LIBRARY_PATH=$HOME/DeepSpeech/CUDA/lib64/:$LD_LIBRARY_PATH`` in your ~/.profile or others
+ * ``echo "$HOME/DeepSpeech/CUDA/lib64/" > /etc/ld.so.conf.d/cuda.conf && ldconfig`` (as root)
+
+# Extra changes
+
+If building on some recent systems, you might get strange errors about ``<string>``
+during the build. It seems to be related to recent GCC. Please apply the following patch:
+```
+diff --git a/third_party/gpus/crosstool/CROSSTOOL.tpl b/third_party/gpus/crosstool/CROSSTOOL.tpl
+index 3ce6b74..be726ff 100644
+--- a/third_party/gpus/crosstool/CROSSTOOL.tpl
++++ b/third_party/gpus/crosstool/CROSSTOOL.tpl
+@@ -54,6 +54,7 @@ toolchain {
+   # Use "-std=c++11" for nvcc. For consistency, force both the host compiler
+   # and the device compiler to use "-std=c++11".
+   cxx_flag: "-std=c++11"
++  cxx_flag: "-D_FORCE_INLINES"
+   linker_flag: "-lstdc++"
+   linker_flag: "-B/usr/bin/"
+```
+
+# Configure step
+## CPU Only:
+* ``echo "" | PATH=$HOME/DeepSpeech/bin/:$PATH PYTHON_BIN_PATH=$HOME/DeepSpeech/tf-venv/bin/python PYTHONPATH=$HOME/DeepSpeech/tf-venv/lib/python2.7/ TF_NEED_GCP=0 TF_NEED_HDFS=0 TF_NEED_CUDA=0 ./configure``
```

Comment:
You don't have a /usr/bin/gcc-4.9 symlink ? Using it during the configure step should not require any change of CC. But it shows that I forgot the GCC COMPILER PATH variable for CPU Only :)